tests/runtime/basic

```diff
@@ -134,6 +134,22 @@ RUN {{BPFTRACE}} -l "rawtracepoint:*"
 EXPECT rawtracepoint:
 TIMEOUT 1
 
+NAME it lists probes in the program
+RUN {{BPFTRACE}} -l -e 'kfunc:vmlinux:vfs_* { exit(); }'
+EXPECT kfunc:vmlinux:vfs_read
+TIMEOUT 5
+
+NAME it lists uprobes in the program
+RUN {{BPFTRACE}} -l -e 'uretprobe:*:uprobeFunction* { exit(); }' -p {{BEFORE_PID}}
+EXPECT uretprobe:[\S]+uprobe_test:uprobeFunction1
+TIMEOUT 5
+BEFORE ./testprogs/uprobe_test
+
+NAME it lists multiple probes in the program
+RUN {{BPFTRACE}} -l -e 'hardware:cache-misses:10 { exit(); } tracepoint:xdp:mem_connect { exit(); }'
+EXPECT hardware:cache-misses:\ntracepoint:xdp:mem_connect
+TIMEOUT 5
```

Comment:
Given this PR reduces the scope of the output compared to previous behavior, I don't think these tests catch any future regressions, right? We need some negative regex tests as well

Reply:
The only one that is superfluous is the first one as `-l` by itself doesn't show uprobes and the third test shows these two probes directly following each other in the list, which doesn't happen with just `-l`. But let me see if I can't improve these a bit more.

Reply:
Ah got it. Maybe some kind of EXPECT_NOT directive?

Reply:
Yeah I think EXPECT_NOT was mentioned on another PR. I'll add it soon if someone doesn't beat me to it.